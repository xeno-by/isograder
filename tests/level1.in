* Section [/1]
*** Subsection [/1]