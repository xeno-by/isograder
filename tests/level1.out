EXIT CODE:
1

STDOUT:

STDERR:
error parsing the correction sheet:
line 2: invalid section level: expected **, actual ***
*** Subsection [/1]
^

